AUTHORS.txt

```diff
@@ -30,16 +31,15 @@ Former members of the AiiDA team
 * Riccardo Sabatini (2013-2015)
 
 
-Contributors
-############
-We would like to thank the following people for their major contibutions:
+## Contributors
+
+We would like to thank
 
 * Dominik Gresch (ETHZ): for the caching mechanism and contributions to the workflow engine
 * Tiziano Müller (ETHZ, Switzerland): for the first implementation of
```

Comment:
```suggestion
* Tiziano Müller (ETHZ): for the first implementation of the plugin system and making the original code python 3 compatible
```

Reply:
Last I've known @dev-zero was at UZH, not ETHZ.